library/psa_crypto.c

```diff
@@ -1126,7 +1128,7 @@ psa_status_t psa_destroy_key(mbedtls_svc_key_id_t key)
 
 exit:
     status = psa_wipe_key_slot(slot);
-    /* Prioritize CORRUPTION_DETECTED from wiping over a storage error */
+    /* Prioritize an error from wiping over a storage error */
```

Comment:
We don't need this change if we change `psa_key_slot_state_transition()` to return `PSA_ERROR_CORRUPTION_DETECTED`.